docs/source/connecting.rst

```diff
@@ -0,0 +1,59 @@
+Getting connected to Tor
+========================
+
+When OnionShare starts, it will present a screen asking you to connect to the Tor network.
+
+.. image:: _static/screenshots/autoconnect-welcome-screen.png
+
+You have several choices. You can click Connect to Tor to begin the connection process. If there are no problems with your network, including any attempts to block your access to the Tor network, this should hopefully work the first time.
+
+If you want to manually configure Bridges or other Tor settings before you connect, you can click 'Network Settings'.
+
+Automatic censorship circumvention
+----------------------------------
+
+When OnionShare fails to connect to Tor, it might be because Tor is censored in your country.
+
+If this occurs, a screen will be displayed that will offer you to either 'Use a Bridge' (automatically choosing one for you), or 'Try Again without a Bridge'.
+
+.. image:: _static/screenshots/autoconnect-failed-to-connect.png
+
+What 'Use a Bridge' does
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Bridges are a way to connect to the Tor network via intermediate services that are hopefully not blocked by your network provider (whereas normal connections might be).
+
+Clicking 'Use a Bridge' will temporarily use the `Meek <https://gitlab.torproject.org/legacy/trac/-/wikis/doc/meek/>`_ domain-fronting proxy to make a non-Tor connection from your computer to the Tor Project's Censorship Circumvention API. The use of the Meek proxy is to hide the fact that you are trying to find a way to connect to Tor.
+
+Before clicking 'Use a Bridge', you can choose either 'Automatically determine my country from my IP address' or 'Manually select my country'.
+
+If you choose the former option, the Censorship Circumvention API will consider your IP address (yes, your real IP address) to determine what country you might reside in.
+
+Based on the country information, the API will try to automatically find bridges that suit your location.
+
+.. image:: _static/screenshots/autoconnect-trying-to-resolve-connectivity-issues.png
+
+If you choose to manually select a country, OnionShare will still make a request to the Censorship Circumvention API, but will tell the API the country code. The API will then try to fetch bridges for that country that you specified.
```

Comment:
Maybe we can mention here that in this scenario the API doesn't check your real IP address which makes it a little better for some folks.

Reply:
The thing is that the API knows the IP address either way, it just doesn't evaluate it against a geoIP database. We have to trust that the TP's API doesn't log the IP or do anything else with it, but we can't prove it, so I don't want to give any misleading assurance

Reply:
Agreed